setup.py

```diff
@@ -42,7 +42,7 @@
     license="Apache 2",
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
-    install_requires=["pytket ~= 1.26"],
+    install_requires=["pytket ~= 1.26", "networkx >= 2.8"],
```

Comment:
```suggestion
    install_requires=["pytket ~= 1.26", "networkx ~= 3.0"],

Reply:
I would be careful with allowing 4.0 here, do you need 2.x here, or should we just ask for 3.x to be installed? (see suggestion)

Reply:
Yep, it works 👍